frontend/libs/studio-components/src/components/StudioTable/StudioTable.test.tsx

```diff
@@ -0,0 +1,83 @@
+import type { ForwardedRef } from 'react';
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import type { StudioTableProps } from './';
+import { StudioTable } from './';
+import { testRefForwarding } from '../../test-utils/testRefForwarding';
+import { testCustomAttributes } from '../../test-utils/testCustomAttributes';
+import { testRootClassNameAppending } from '../../test-utils/testRootClassNameAppending';
+
+// Test data:
+const headContents = ['Header 1', 'Header 2'];
+const bodyContents = [
+  ['Row 1, column 1', 'Row 1, column 2'],
+  ['Row 2, column 1', 'Row 2, column 2'],
+];
+
+describe('StudioTable', () => {
+  it('Renders a table', () => {
+    renderTable();
+    getTable();
```

Comment:
This isn’t a big issue, but we had a team discussion some time ago and agreed to avoid using "get" variants as assertions. Instead, we’re following the recommendation from this blog post:
[Common mistakes with React Testing Library](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-get-variants-as-assertions).

Reply:
Great article you found there, @framitdavid. I don't have any strong opinions about this; I find myself using both approaches. Mostly when I avoid `toBeInTheDocument`, it's to keep the line length within the limit, but once again that doesn't apply here.

Maybe we should create a page on Confluence where we gather all the rules that we agree on with the team?

Reply:
Yes, let's create a document at Confluence now, since we have access now. 👏

Reply:
We do not have access to Confluence yet... Let's make it a follow-up based on action point from retro. Resolving this issue.